chromadb/utils/embedding_functions.py

```diff
@@ -144,3 +144,25 @@ def __call__(self, texts: Documents) -> Embeddings:
             ]
             for text in texts
         ]
+
+class GoogleVertexEmbeddingFunction(EmbeddingFunction):
+    def __init__(
+        self,
+        api_key: str,
+        model_name: str = "textembedding-gecko-001",
+        project_id: str = "cloud-large-language-models",
```

Comment:
is the `project_id` something the user provides? google specifies? what are the available correct fields here?